backend/templates/v2/ngo-account/redirections/main.html

```diff
@@ -40,7 +44,15 @@ <h1 class="text-base font-semibold text-gray-900">
     <hr class="w-full border-gray-200">
 
     {% with form_name="redirections_search_form" %}
-      <form action="" method="GET" id="{{ form_name }}" name="{{ form_name }}">
+        <form
+          hx-get={{ request.path }}
+          hx-target="#listings"
+          hx-select="#listings"
+          hx-trigger="input delay:500ms from:#searchBox_redirections_search, submit"
+          method="GET"
+          id="{{ form_name }}"
+          name="{{ form_name }}"
+        >
```

Comment:
As I mentioned, if you want to continue with this you'll have to work with Alpine JS

There's [debounce for input delays](https://alpinejs.dev/directives/model#debounce) in the model directive and other directives to help.